pkg/armhelpers/utils/util.go

```diff
@@ -26,9 +26,9 @@ const (
 	// here there are 2 capture groups
 	//  the first is the agent pool name, which can contain -s
 	//  the second group is the Cluster ID for the cluster
-	vmssNamingFormat       = "^[0-9a-zA-Z]+-(.+)-([0-9a-fA-F]{8})-vmss$"
-	vmssAgentPoolNameIndex = 1
-	vmssClusterIDIndex     = 2
+	VMSSNamingFormat       = "^[0-9a-zA-Z]+-(.+)-([0-9a-fA-F]{8})-vmss$"
+	VMSSAgentPoolNameIndex = 1
+	VMSSClusterIDIndex     = 2
```

Comment:
these should probably be constants... (not added by your PR but in case you want some extra credit :P )